models.py

```diff
@@ -411,7 +413,7 @@ def _annotate_first_of_milestones(self, feature_list, version=None):
         IMPLEMENTATION_STATUS[NO_ACTIVE_DEV],
         IMPLEMENTATION_STATUS[PROPOSED],
         IMPLEMENTATION_STATUS[IN_DEVELOPMENT],
-        IMPLEMENTATION_STATUS[DEPRECATED]
+        IMPLEMENTATION_STATUS[DEPRECATED],
```

Comment:
Did you mean to add a comma to the last item in an array?

Reply:
Yep, for consistency with the surrounding code. In JS code the linter even requires commas at the end of lists like this (not something I personally agree with, but consistency outweighs my personal feelings here :).